id_minter/src/main/scala/uk/ac/wellcome/platform/idminter/database/IdentifiersDao.scala

```diff
@@ -16,22 +16,22 @@ class IdentifiersDao @Inject()(db: DB, identifiers: IdentifiersTable)
 
   implicit val session = AutoSession(db.settingsProvider)
 
-  def findSourceIdInDb(miroId: String): Future[Option[Identifier]] =
+  def lookupMiroID(miroID: String): Future[Option[Identifier]] =
     Future {
       blocking {
-        info(s"About to search for MiroID $miroId in Identifiers")
+        info(s"About to search for MiroID $miroID in Identifiers")
         val i = identifiers.i
         withSQL {
-          select.from(identifiers as i).where.eq(i.MiroID, miroId)
+          select.from(identifiers as i).where.eq(i.MiroID, miroID)
         }.map(Identifier(i)).single.apply()
       }
     } recover {
       case e: Throwable =>
-        error(s"Failed getting MiroID $miroId in DynamoDB", e)
+        error(s"Failed getting MiroID $miroID in Identifiers", e)
         throw e
     }
 
-  def saveIdentifier(identifier: Identifier): Future[Unit] = {
+  def saveIdentifier(identifier: Identifier): Future[Int] = {
```

Comment:
That’s what the `apply()` method returns, I didn't have time to look up why. Felt better than just dropping it and returning an empty `Unit`.